torch/_torch_docs.py

```diff
@@ -214,68 +214,43 @@ def merge_dicts(*dicts):
 """.format(**common_args))
 
 add_docstr(torch.add, r"""
-add(input, other, *, out=None) -> Tensor
+add(input, other, *, alpha=1, out=None) -> Tensor
 
-Adds the scalar :attr:`other` to each element of the input :attr:`input`
-and returns a new resulting tensor.
+Adds :attr:`other`, scaled by :attr:`alpha`, to :attr:`input` elementwise.
```

Comment:
nit: remove `elementwise` to be consistent with `sub`

Reply:
Sure